Fabric.Authorization.Persistence.SqlServer/Stores/EDW/ISecurityContext.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Catalyst.Security.Services
```

Comment:
Namespace doesn't match folder structure?

Reply:
What's the reasoning behind the name `ISecurityContext` vs. `IMetadataDatabaseContext` or something like that?